src/Import/ImportFinder.php

```diff
@@ -11,7 +11,7 @@
 /**
  * Import finder which uses ImportCollectorInterface to find imports.
  */
-final class ImportFinder implements ImportFinderInterface
+class ImportFinder implements ImportFinderInterface
```

Comment:
Same as the other one, should probably not become an extension point. People _will_ extend and break it.